packages/addons-website/scss/components/_swatch-palette-widget.scss

```
@@ -0,0 +1,89 @@
+@import '../prefix';
+
+.#{$prefix}--swatch-palettes {
+  display: flex;
+  flex-direction: column;
+  .#{$prefix}--swatch-palette {
+    float: left;
+    width: 100%;
+    margin-bottom: 2rem;
+    .#{$prefix}--swatch {
+      padding: 1rem;
+      font-family: IBM Plex Mono,Menlo,DejaVu Sans Mono,Bitstream Vera Sans Mono,Courier,monospace;
+      .#{$prefix}--swatch-name {
+        text-transform: capitalize;
+      }
+      .#{$prefix}--swatch-value {
+        float: right;
+      }
+    }
+    
+    @include breakpoint('md') {
+      width: calc(50% - 1px);
+      margin-right: 1px; 
```

```suggestion
      margin-right: rem(1px); 
```

can make this update anywhere we have px values

---

The margin is directly related to the width calc on the line before. Either they should both remain 1 px or they should both be rem(1px) - not sure that would work within the calc function

---

oh ok, that's fine to leave this then